.changeset/popular-seas-yawn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react-brand': patch
```

Comment:
```suggestion
'@primer/react-brand': minor
```

Reply:
Updated this, but just want to double-check my understanding.

I know we use minor for breaking changes, but do we also default to minor when adding a new component too?

Reply:
> but do we also default to minor when adding a new component too?

👍 Yup breaking changes and new components...